src/test/java/org/sagebionetworks/web/client/SynapseJavascriptClientTest.java

```diff
@@ -917,8 +882,8 @@ public void testDeleteRequestsCannotBeCancelled() throws RequestException {
 		when(mockRequest2.isPending()).thenReturn(true);
 		when(mockGwt.getCurrentURL()).thenReturn(currentUrl);
 
-		client.logout();
-
+		client.deleteEntityById("syn111", null);
```

Comment:
This test is verifying that a DELETE call cannot be canceled.  logout() has been removed from the codebase, so use another DELETE call.

Reply:
We want make sure that the delete does not added to the list of cancellable requests client.getRequests().